README.md

```diff
@@ -42,10 +42,12 @@ If you wish to try out early Sanitizer implementations, the
 
 ## Explainer
 
-The core API of the Sanitizer is rather simple: There are `Sanitizer` objects,
-and they have a `.sanitize` method that produces a `DocumentFragment` from a
-given text string. Or from another `DocumentFragment`, or even from a whole
-`Document`.
+The core API of the Sanitizer is rather simple: There are `Sanitizer` objects
+which encapsulate the information of what the sanitizer is supposed to do,
+and there is an `Element.setHTML` method which takes a string and (optionally)
+an options dictionary with a `"sanitizer"` key and a `Sanitizer` instance as
+a value. `.setHTML` will parse and sanitize the string it receives, and will
+update the children of the Element it is called on with the result.
```

Comment:
Quite the long sentences. Can we cut the complexity by making it grow gradually?
How about something along those lines:
* This API prevents XSS, which means sanitizing a tree of DOM nodes
* There's a safe default that states which should be removed and kept
* if all you care about is XSS, you may just use `someElement.setHTML(untrusted_input)`
* if you need to customize, use a Sanitizer constructor to hold the configuration and pass it in

Would that make sense?